kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/order/CreateExternalOrderCall.java

```diff
@@ -100,6 +101,36 @@ public void run() {
 		});
 	}
 
+	private void sendOrderCreationFailedEvent(ApiException exception) {
+		if (openOrder != null) {
```

Comment:
need to send even if openOrder !=null

Reply:
we have a problem - if the SpendOrderCreationFailed it means that we don't have openOrder == null by definition ????